pynest/nest/lib/hl_api_nodes.py

```diff
@@ -83,12 +83,13 @@ def Create(model, n=1, params=None, positions=None):
     model_deprecation_warning(model)
 
     if positions is not None:
+        spatial = getattr(nest.NestModule, "spatial")
         # We only accept positions as either a free object or a grid object.
-        if not isinstance(positions, (nest.spatial.free, nest.spatial.grid)):
-            raise TypeError('`positions` must be either a nest.spatial.free object or nest.spatial.grid object')
+        if not isinstance(positions, (spatial.free, spatial.grid)):
+            raise TypeError('`positions` must be either a spatial.free object or spatial.grid object')
```

Comment:
I would leave `nest.spatial.` in the error message since this is the correct module path for users.

Reply:
Agreed.
```suggestion
            raise TypeError('`positions` must be either a nest.spatial.free or a nest.spatial.grid object')
```